dnf/cli/commands/__init__.py

```diff
@@ -217,8 +217,26 @@ class ListCommand(InfoCommand):
 
     def run(self):
         self.cli._populate_update_security_filter(self.opts)
-        return self.base.output_packages('list', self.opts.packages_action,
-                                         self.opts.packages)
+        try:
+            self.base.output_packages('list', self.opts.packages_action,
+                                      self.opts.packages)
+        except dnf.exceptions.Error as e:
+            logger.warning(e)
+
+        print("Modules")
+        if self.opts.packages_action == "installed":
+            # show *enabled* modules
+            if self.opts.packages:
+                for pkg in self.opts.packages:
+                    print(self.base.repo_module_dict.get_brief_description_enabled(pkg))
```

Comment:
What about to create complete output in list then it can be printed by single print?